docs/basic/attributes.md

```diff
@@ -291,6 +334,32 @@ Pada kode di atas, attribute name `allow` digunakan pada 3 tempat:
 
 Dengan penambahan 3 attribute di atas program akan tereksekusi tanpa warning.
 
+Selain itu ada juga attribute `forbid`.
```

Comment:
this is also great to have. I'm adjusting the content and removing the `unsafe_code` because we haven't go into it. this chapter already big enough for beginner to grasp all of the content. we will talk about `unsafe_code` on separate chapter